docs/backends/lmdb.rst

```diff
@@ -0,0 +1,59 @@
+LMDB backend
+============
+
+* Native: Yes
+* Master: Yes
+* Slave: Yes
+* Superslave: No
+* Case: All lower
+* DNSSEC: Yes 
+* Disabled data: No
+* Comments: No
+* Module name: lmdb
+* Launch name: ``lmdb``
+
+Enabling the backend
+--------------------
+
+When building PowerDNS yourself, append ``lmdb`` to ``--with-modules`` or ``--with-dynmodules``. It is expected that most pre-built packages contain this backend or be separately installable.
+
+
+Settings
+--------
+
+.. _setting-lmdb-filename:
+
+``lmdb-filename``
+^^^^^^^^^^^^^^^^^
+
+Path to the LMDB file (e.g. */var/spool/powerdns/pdns.lmdb*)
+
+.. warning::
+  On systemd systems, 
+  When running PowerDNS via the provided systemd service file, `ProtectSystem <http://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=>`_ is set to ``full``, this means PowerDNS is unable to write to e.g. ``/etc`` and ``/home``, possibly being unable to write to the LMDB database.
+
+.. _setting-lmdb-shards:
+
+``lmdb-shards``
+^^^^^^^^^^^^^^^^^
+
+Records database will be split into this number of shards e.g. lmdb-shards=64
+
+.. _setting-lmdb-sync-mode:
+
+``lmdb-sync-mode``
+^^^^^^^^^^^^^^^^^
+
+Synchronisation mode: nosync, nometasync, mapasync
+
+* ``nosync``: Don't flush systems buffers to disk when committing a transation.
+  This means a system crash can corrupt the database or lose the last transactions if buffers are not yet flushed to disk.
+* ``nometasync``: Flush the data on a commit. Slightly faster  than  doing a full sync, but can potentially lose the last committed transaction if the operating system crashes.
```

Comment:
Am I correct that a full sync is not possible? So the DB is always at risk when the system crashes? If so, is there a recovery procedure? Should be documented if there's one.

Reply:
there is indeed no "sync" option, although the "nometasync" is way better than e.g. nosync, as it won't corrupt the whole DB, but just lost the last committed transaction. 
I'll change the description to make it more clear

Reply:
@franklouwers now fixed - it is possible to have a full "sync" when you specified "lmdb-sync-mode=" - which was not really intuitive.
There's now a "lmdb-sync-mode=sync

Also, I have clarified that the default sync mode is mapasync